app/views/donations/index.html.erb

```diff
@@ -5,22 +5,24 @@
         <% content_for :title, "Donations - #{current_organization.name}" %>
         <h1>
           Donations
-          <small>for <%= current_organization.name %></small>
```

Comment:
Why was this page changed at all? It seems like you're not changing how the above methods are being called, no?

Reply:
Sorry this is a linting thing. I'll get a local env file to disable it.